File: DSA/Stuti_Jain_590012149/Day_18_Question_1.c
#include <stdio.h>
void ssort(int arr[], int n){
    for(int i = 0; i < n - 1; i++){
        int min = i;
        for(int j = i + 1; j < n; j++){
            if(arr[j] < arr[min]){
                min = j;
            }
        }
        int temp = arr[i];
        arr[i] = arr[min];
        arr[min] = temp;
    }
}

void output(int arr[], int n){
    for(int i = 0; i < n; i++){
        printf("%d ", arr[i]);
    }
    printf("\n");
}

int main(){
    int n;
    printf("Enter number of elements: ");
    scanf("%d", &n);

    int arr[n];
    printf("Enter the elements: ", n);
    for(int i=0; i<n; i++){
        scanf("%d", &arr[i]);
    }

    ssort(arr, n);

    printf("");
    output(arr, n);

    return 0;
}

File: DSA/PranyaGupta_500126032/Day18Question1.py
class Solution:
    def Selectionsort(self, arr):
        n = len(arr)
        j=0
        while j<n-1:
            minele = arr[j]
            ind = j
            for i in range(j,n):
                if arr[i]< minele:
                    minele = arr[i]
                    ind = i
                    temp = arr[j]
                    arr[j] = arr[ind]
                    arr[ind] = temp
            j+=1
        return arr              




def main():
    user_input = input("Enter elements of the array (with space in between): ")
    arr = list(map(int, user_input.strip().split()))
    sol = Solution()
    result = sol.Selectionsort(arr)
    print(result)
if __name__ == "__main__":
    main()